src/main/resources/kafka-topics.json

```diff
@@ -0,0 +1,9 @@
+{
+  "topics": [
+    {
+      "name": "inventory.instance",
+      "numPartitions": 1,
+      "replicationFactor": 1
```

Comment:
Does this mean that all messages will be published to the same topic irrespective of tenant?

Reply:
Yes.